test/evergreen.yml

```diff
@@ -1266,6 +1266,37 @@ buildvariants:
     - name: unit-test-bucket06
     - name: unit-test-bucket07
 
+- name: rhel80
+  display_name: RHEL 8.0
+  run_on:
+  - rhel80-test
+  expansions:
+    test_env_vars: PATH=/opt/mongodbtoolchain/v3/bin:$PATH LD_LIBRARY_PATH=$(pwd)/.libs top_srcdir=$(pwd)/.. top_builddir=$(pwd)
+    smp_command: -j $(grep -c ^processor /proc/cpuinfo)
+    configure_env_vars: CC=/opt/mongodbtoolchain/v3/bin/gcc CXX=/opt/mongodbtoolchain/v3/bin/g++ PATH=/opt/mongodbtoolchain/v3/bin:$PATH
+    make_command: PATH=/opt/mongodbtoolchain/v3/bin:$PATH make
+  tasks:
+    - name: compile
+    - name: lang-python-test
+    - name: make-check-test
+    - name: unit-test
+    - name: fops
+
+- name: rhel80-python3
+  display_name: RHEL 8.0 (Python3)
+  run_on:
+  - rhel80-test
+  expansions:
+    test_env_vars: PATH=/opt/mongodbtoolchain/v3/bin:$PATH LD_LIBRARY_PATH=$(pwd)/.libs top_srcdir=$(pwd)/.. top_builddir=$(pwd)
+    smp_command: -j $(grep -c ^processor /proc/cpuinfo)
+    configure_env_vars: CC=/opt/mongodbtoolchain/v3/bin/gcc CXX=/opt/mongodbtoolchain/v3/bin/g++ PATH=/opt/mongodbtoolchain/v3/bin:$PATH
+    configure_python_setting: PYTHON=python3
+    make_command: PATH=/opt/mongodbtoolchain/v3/bin:$PATH make
+    python_binary: python3
+  tasks:
+    - name: compile
+    - name: unit-test
+
```

Comment:
As the ubuntu build variant has covered Python2 tests and Python2 is going to EOL support, I'd suggest we make Python3 the default and only Python testing for the new rhel80 build variant - i.e. merging this `rhel80-python3` build variant with `rhel80`.